thirdeye-ui/src/app/components/home-page/recent-anomalies/recent-anomalies.component.tsx

```diff
@@ -108,29 +110,43 @@ export const RecentAnomalies: FunctionComponent = () => {
                         </>
                     }
                 >
-                    <Table size="small">
-                        <TableHead>
-                            <TableRow>
-                                <TableCell>Anomaly ID</TableCell>
-                                <TableCell>Alert Name</TableCell>
-                                <TableCell>Metric</TableCell>
-                                <TableCell>Started</TableCell>
-                                <TableCell>Ended</TableCell>
-                                <TableCell>Deviation</TableCell>
-                                <TableCell />
-                                <TableCell />
-                            </TableRow>
-                        </TableHead>
-                        {anomaliesToDisplay.length > 0 &&
-                            anomaliesToDisplay.map((anomaly) => {
+                    {anomaliesToDisplay.length > 0 ? (
+                        <Table size="small">
+                            <TableHead>
+                                <TableRow>
+                                    <TableCell>Anomaly ID</TableCell>
+                                    <TableCell>Alert Name</TableCell>
+                                    <TableCell>Metric</TableCell>
+                                    <TableCell>Started</TableCell>
+                                    <TableCell>Ended</TableCell>
+                                    <TableCell>Deviation</TableCell>
+                                    <TableCell />
```

Comment:
Aww dang, looks like missed using i18n for these, can you also add entries to that as well. As well as the message in the no data state